config/schema/apigee_m10n.schema.yml

```diff
@@ -0,0 +1,30 @@
+apigee_m10n.rate_plan.config:
+  type: config_object
+  label: 'Rate plan settings'
+  mapping:
+    product_rate_plan_view_mode:
+      type: string
+      label: 'View mode to use when viewing rate plans within packages.'
+
+apigee_m10n.subscription.config:
+  type: config_object
+  label: 'Subscription settings'
+  mapping:
+    subscribe_form_title:
```

Comment:
Suggestion: I would make the  title and the label `type: label` as unlike a `string` a `label` is translatable.

Reply:
Nice, I'll make that change for config items that are used for display.